src/grid/interpolate.jl

```diff
@@ -98,3 +99,72 @@ linear interpolation of data(x, y)
     gx = (g0 * dy1 + g1 * dy0) / (dx0 + dx1) / (dy0 + dy1)
     return gx
 end
+
+
+"""
+testInterpolation1D(func, grid1, grid2)
+
+Test interpolation on grid1 with function func, compare results with func(grid2).
+Return max deviation and std of deviation.
+
```

Comment:
use the standard format for the comments:

#Arguments

-  `func`: function needs to be interpolated
-  `grid1`: grid to test
-  `grid2`: grid to test